worker/src/RTC/Router.cpp

```diff
@@ -660,6 +661,11 @@ namespace RTC
 			// Clone only happens if needed.
 			std::shared_ptr<RTC::RtpPacket> sharedPacket;
 
+#ifdef MS_LIBURING_ENABLED
+			// Activate liburing usage.
+			DepLibUring::SetActive();
+#endif
```

Comment:
This is very confusing. Is it enabled because a macro/define is given? Or is it supported? And what does active mean?

Reply:
@ibc, let me know if this is clear.

Reply:
So we use "enabled" to say that it's included in the binary and "supported" to say that it's effectively available in runtime? That's confusing, even more when we ALSO have "active", meaning that it's "enabled" in the binary, "supported" in runtime and in use right now.

Reply:
I would rename MS_LIBURING_ENABLED to MS_LIBURING_SUPPORTED and then have a static DepLibUring::IsSupported() that returns true if that macro is given and kernel version is good. Anyway, with my other proposal the macro is gonna be used just within DepLibUring hpp file (which now should be always included) and the rest of the code would just check that static method.

Reply:
I'll rename `MS_LIBURING_ENABLED` by `MS_LIBURING_SUPPORTED` .